### ISSUE TITLE
Add a new committer to this project?

### ISSUE THREAD

**Issue by rmosolgo** [reactjs/react-rails#785]
I recently got a new job that got me out of the react.js business (😿 ) so I can't give this project as much attention as I used to. I'm happy to help out from time to time or review changes to old code, but I can't really push react-rails into the future. 

Is anyone interested in taking over this project?

**Comment by rmosolgo** [comment]
cc @BookOfGreg if you're interested I'd be happy to vouch for you (I can't add committers, but I can ask a real FB-er to do it)

**Comment by vipulnsward** [comment]
Thanks @rmosolgo for taking it upto here! ❤️

**Comment by SemanticMike** [comment]
+1 for @BookOfGreg

**Comment by BookOfGreg** [comment]
Apparently half my workplace wants me to take this one on.
Yep I'll be glad to give it a go. My interest is mostly around the Webpack integration + performance as it affect my day job so it may be worth seeking someone who relies on the sprockets side of things in addition.

**Comment by rmosolgo** [comment]
Glad to hear it! I'm happy to keep an eye on the Sprockets component, that's _my_ sweet spot. 

@zpao, do you mind adding @BookOfGreg to this project? 

Also @BookOfGreg do you already have a RubyGems profile? (eg https://rubygems.org/profiles/rdmosolgo)

**Comment by BookOfGreg** [comment]
@rmosolgo Glad we can work to our strengths 👍 
https://rubygems.org/profiles/BookOfGreg

**Comment by hrishimittal** [comment]
@rmosolgo Robert, thanks so much for all your amazing work. Good luck with the new job!

**Comment by BookOfGreg** [comment]
@rmosolgo Is @zpao the main owner of this gem? Who should I know about when working in this area?

**Comment by rmosolgo** [comment]
Sure, how bout a bit of history :) 

In October 2014, I rewrote the front end of an app from Batman.js (RIP) to React, using this project. But I saw there were a lot of unanswered questions and unreviewed PRs, so I started answering and reviewing. So zpao noticed and made me a maintainer, and since then I've been "at the helm". 

But there hasn't been a road map, I've just maintained the same features, plus adding a few extension points here and there to support peoples' requests. 

I like this project for the same reasons you mentioned a while back: it's not tightly coupled with a build system or Flux implementation. It's Rails first, with React on top, not vice-versa. So I didn't really have a vision for other features, I was (and am) happy with the feature set it has. 

So to sum up: 

>  pick off what I can when I've got time

That's basically what I've been up to. And keeping up with Rails stuff, like new Turbolinks versions, Webpacker, etc. There might be more to do, but I haven't had the time to do it lately 😬

**Comment by rmosolgo** [comment]
Btw @BookOfGreg Sorry, I asked hte wrong question. To add you to rubygems, I need the _email_ associated with your account there, can you share it? If you don't want to post it here, feel free to send it to me by the email address on my github profile.

**Comment by BookOfGreg** [comment]
Ah, my inexperience is showing! Email is in my Github profile now. Also added my company so you can see who the sponsor has been for my time spent on this so far.

I like the attitude the project and yourself have taken so far. This is already the most code I've contributed to open source so far, mostly documentation on other projects so I hope I can make folk happy with what we're doing here.

**Comment by rmosolgo** [comment]
:+1: Thanks for posting that, I added you with `$ gem owner --add ...`. So, you should be able to release with the [newly-added release guides](https://github.com/reactjs/react-rails/blob/master/README.md#development). Happy to hook you up on NPM too if you want to make an account, I forgot the UJS gets published there now, too :P 

I pinged @zpao on twitter to see if he can give you the keys to the repo. 

FWIW react-rails has been my first foray into OSS too :) 

>  make folk happy

That's something I've enjoyed about maintaining react-rails. It's a high-visibility project, so when you get something going (like the upcoming Webpacker 3 release), people are happy about it 😊 

But I think a lot of people are deterred by the complexity of the project, which is required to support the various build tools. So, I was really impressed by your work on Webpacker compatibility!

**Comment by zpao** [comment]
Sorry for my lag in the process. Invite sent so once you accept that @BookOfGreg, you're in. Welcome to open source maintenance!

![giphy - welcome aboard](https://user-images.githubusercontent.com/8445/30887727-8781b232-a2d1-11e7-9452-8969bebeb24b.gif)

@rmosolgo I hope you won't be a stranger - you've done an amazing job keeping this project going.

**Comment by BookOfGreg** [comment]
Going to close this for now, if anyone comes across this, we are always going to be happy to have more maintainers and contributors helping us drive the cutting edge of tech!
